src/frontend/src/binder/statement.rs

```diff
@@ -68,3 +69,18 @@ impl Binder {
         }
     }
 }
+
+pub(crate) trait RewriteExprsRecursive {
+    fn rewrite_exprs_recursive(&mut self, rewriter: &mut impl ExprRewriter);
+}
```

Comment:
I think modifying bound statement maybe inappropriate since we need to reuse the statement for each coming data. It would be better to return a new statement after rewrite.

Reply:
I find that actually we don't need to return a new statement. We will store the statement into a HashMap first. And get the statement(clone) to bind the parameter when we get the bind message. So that we always have a originally version in the HashMap.